source/_integrations/leviosa_shades.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Leviosa Motor Blinds Zone hub
+description: Instructions on how to integrate the Leviosa Zone hub into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_release: 2021.4
+ha_domain: leviosa_shades
+ha_codeowners:
+  - '@altersis'
+ha_config_flow: true
+ha_platforms:
+  - cover
+---
+
+This integration allows you to control [Leviosa Zone hubs](https://leviosashades.com/products/leviosa-zone) from [Leviosa Motor Shades](https://leviosashades.com/).
+
+{% include integrations/config_flow.md %}
+- Each Leviosa Zone manages up to 6 groups of shades.
```

Comment:
Possibly a title above this makes it a bit clearer, what you read underneath.

Considering it's a bit of a "keep in mind" piece 😄